src/js/modules/facetedsearch/index.ts

```diff
@@ -65,10 +65,14 @@ const initSliders = () => {
     const sliderValues = JSON.parse(<string>container.dataset.sliderValues);
 
     if(!container.noUiSlider) {
+      const searchFilters = <HTMLElement>document.querySelector('.search-filters');
```

Comment:
Please use the selector-map.ts file, it's easier for theme developer and maintainers to deal with it